data/org.pantheon.switchboard.parental-controls.policy.in

```diff
@@ -7,8 +7,8 @@
   <vendor_url>https://elementary.io</vendor_url>
 
   <action id="org.pantheon.switchboard.parental-controls.administration">
-    <description gettext-domain="@GETTEXT_PACKAGE@">Manage parental control settings</description>
-    <message gettext-domain="@GETTEXT_PACKAGE@">Authentication is required to change parental control settings</message>
+    <description gettext-domain="@GETTEXT_PACKAGE@">Manage Screen Time &amp; Limits settings</description>
```

Comment:
Is the string "settings necessary here? It seems like it would be just as clear and sound more natural to do
```suggestion
    <description gettext-domain="@GETTEXT_PACKAGE@">Manage Screen Time &amp; Limits</description>
```